src/main/java/dao/MysqlCarDAO.java

```diff
@@ -0,0 +1,181 @@
+package dao;
+
+import entity.Car;
+import entity.CarColor;
+import entity.CarType;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MysqlCarDAO implements ICarDAO {
+
+    private Connection connection;
+    private final String BASEURL = "jdbc:mysql://localhost:3306/cars";
+
+
+    public MysqlCarDAO(String user, String password) {
+
+        try {
+            Class.forName("com.mysql.jdbc.Driver");
+            connection = DriverManager.getConnection(BASEURL, user, password);
+        } catch (ClassNotFoundException e) {
+            e.printStackTrace();
+        } catch (SQLException e) {
+            try {
+                connection = DriverManager.getConnection("jdbc:mysql://localhost:3306", user, password);
+                connection.createStatement().execute("CREATE DATABASE cars;");
```

Comment:
IMO, better to connect to existing database. Because second run of this program will be failed.

Reply:
maybe just throw sql exception